x/staking/module.go

```diff
@@ -143,7 +143,7 @@ func (am AppModule) Route() string {
 
 // NewHandler returns an sdk.Handler for the staking module.
 func (am AppModule) NewHandler() sdk.Handler {
```

Comment:
Maybe just eliminate the customary handler?

Reply:
we need it since it isn't a wrapper type